### PR TITLE
Restify publish controller fix

### PR DESCRIPTION
**Pull request by CaReS0107**

### CODE REVIEW

File: src/Commands/stubs/Auth/RegisterController.stub [hunk 1]
```diff
@@ -18,12 +18,15 @@ class RegisterController extends Controller
             'password' => ['required', 'confirmed'],
         ]);
 
-        User::forceCreate([
+       $user = User::forceCreate([
```
Comment (arthurkirkosa) [inline src/Commands/stubs/Auth/RegisterController.stub:21]: Indentation

File: src/Commands/stubs/Auth/LoginController.stub [hunk 2]
```diff
@@ -29,7 +28,7 @@ class LoginController extends Controller
             abort(401, 'Invalid credentials.');
         }
 
-        $token = Password::createToken($user);
+        $token = $user->createToken('login');
```
Comment (arthurkirkosa) [inline src/Commands/stubs/Auth/LoginController.stub:31]: You can use the same logic as the stub, where you don't create a variable that's not used

File: src/Http/Controllers/Auth/LoginController.php [hunk 3]
```diff
@@ -30,7 +29,7 @@ public function __invoke(Request $request): JsonResponse
             abort(401, 'Invalid credentials.');
         }
 
-        $token = Password::createToken($user);
+        $token = $user->createToken('login');
```
Comment (arthurkirkosa) [inline src/Http/Controllers/Auth/LoginController.php:32]: Same as the other two comments

**Review by arthurkirkosa — APPROVED** [review]

**Comment by binaryk** [comment]
Thank you!

**Pull request merged**